packages/components/src/overlays/Popover/src/Popover.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+    composeClassnameRenderProps,
+    SlotProvider,
+    cssModule,
+    ButtonContext,
+    ButtonGroupContext,
+    ContentContext,
+    FooterContext,
+    TextContext,
+    HeadingContext,
+    LinkContext,
+    useColorSchemeContext, HopperProvider, isNil, isFunction
+} from "@hopper-ui/components";
+import { type StyledComponentProps, useStyledSystem } from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { forwardRef, type ForwardedRef } from "react";
+import {
+    useContextProps,
+    type PopoverProps as RACPopoverProps,
+    type DialogTriggerProps,
+    DialogTrigger,
+    Popover as RACPopover,
+    Dialog
+} from "react-aria-components";
+
+import { PopoverContext } from "./PopoverContext.ts";
+
+import styles from "./Popover.module.css";
+
+export const GlobalPopoverCssSelector = "hop-Popover";
+
+export interface PopoverProps extends StyledComponentProps<RACPopoverProps> {
```

Comment:
Is there some props we should omit, such as arrowBoundaryOffset?

Reply:
it was talking about an "arrow", but we don't have one. could we rename to props to something more appropriate and forward the value to arrowBoundaryOffset?